transactions/update-contract/2023/feb-23/README.md

```diff
@@ -19,6 +19,88 @@ Used this to generate args:
 
 `cat "./FlowIDTableStaking.cdc" | xxd -p -c0 | tr -d '\n'`
 
+Verified using:
+```
+$ cat arguments-update-contract-FlowIDTableStaking-mainnet.json | jq '.[] | .value' | xxd -r -p > temp1.txt
+$ diff temp1.txt FlowIDTableStaking.cdc
+(Should produce no difference)
+```
+
+> Tools is unable to generate the link since the argument content is too large. We will have to do this using the old way using `flow cli`
+
+## Dapper Labs Builds
+
+```sh
+flow transactions build ./transactions/update-contract/2023/feb-23/upgrade-set-candidate.cdc \
+  --config-path flow-staking.json \
+  --network mainnet \
+  --args-json "$(cat "./transactions/update-contract/2023/feb-23/arguments-update-contract-FlowIDTableStaking-mainnet.json")" \
+  --proposer 0x8d0f8df62029f75a \
+  --proposer-key-index 0 \
+  --authorizer 0x8624b52f9ddcd04a \
+  --payer 0x8624b52f9ddcd04a \
+  -x payload \
+  --save ./transactions/update-contract/2023/feb-23/flow-id-table-staking-contract-upgrade-feb-23-unsigned.rlp
+```
+
+## Josh Signs
+
+```sh
+flow transactions sign ./transactions/update-contract/2023/feb-23/flow-id-table-staking-contract-upgrade-feb-23-unsigned.rlp \
+  --config-path flow-staking.json \
+  --signer vishal \
```

Comment:
This should be `josh`, right?

Reply:
yup - i was testing signing the tx and forgot to revert it to josh from vishal